### PR TITLE
Template changes 202404.2.0

### PR DESCRIPTION
**Pull request by srajeshwari-conga**
End-to-end testing has been done.

### CODE REVIEW

File: src/app/modules/checkout/layout/checkout.component.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h4 class="mb-3 text-muted">{{'CART.CREATE_ORDER' | translate}}</h4>
         </div>
       </div>
     </div>
-    <div class="d-lg-none d-md-none d-sm-flex d-flex justify-content-between"> 
+    <div class="d-lg-none d-md-none d-sm-flex d-flex justify-content-between"> <!-- TODO: Added for demo purpose. Need to revisit with UX team-->
```
Comment (vbhan-conga) [inline src/app/modules/checkout/layout/checkout.component.html:26]: TODO needs to be removed, please take care

File: src/app/modules/quotes/quote-create/create-quote.component.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h4 *ngIf="!cart?.Proposald?.Id">{{'COMMON.REQUEST_QUOTE' | translate }}</h4>
                     </button>
                 </div>
             </div>
-            <div class="d-lg-none d-md-none d-sm-flex d-flex justify-content-between">
+            <div class="d-lg-none d-md-none d-sm-flex d-flex justify-content-between"> <!-- TODO: Added for demo purpose. Need to revisit with UX team-->
```
Comment (vbhan-conga) [inline src/app/modules/quotes/quote-create/create-quote.component.html:26]: Please remove TODO statement.

File: src/app/modules/checkout/layout/checkout.component.html (hunk 1 above)
Comment (srajeshwari-conga) [reply src/app/modules/checkout/layout/checkout.component.html]: Changes added.

File: src/app/modules/quotes/quote-create/create-quote.component.html (hunk 2 above)
Comment (srajeshwari-conga) [reply src/app/modules/quotes/quote-create/create-quote.component.html]: Changes added

**Review by vbhan-conga — APPROVED** [review]

**Pull request merged**